cabal-install/Main.hs

```diff
@@ -258,6 +256,7 @@ mainWorker args = topHandler $
       , regularCmd runCommand runAction
       , regularCmd initCommand initAction
       , regularCmd configureExCommand configureAction
+      , regularCmd reconfigureCommand reconfigureAction
```

Comment:
Thanks, this makes sense. Can we add this example to `cabal help reconfigure` and maybe to the manual?